languages/clojure/exercises/concept/vectors/src/vectors.clj

```diff
@@ -0,0 +1,22 @@
+(ns vectors)
+
+(def last-week [0 2 5 3 7 8 4])
+(def birds-per-day [2 5 0 7 4 1])
+
+(defn today [birds]
+  (last birds))
+
+(defn inc-bird [birds]
+  (update birds (dec (count birds)) inc))
+
+(defn day-without-birds? [birds]
+  (pos? (count (filter zero? birds))))
+
+(defn n-days-count [birds n]
+  (reduce + (take n birds)))
+
+(defn busy-days [birds]
+  (count (filter #(>= % 5) birds)))
+
+(defn odd-week? [birds]
+  (= birds [1 0 1 0 1 0 1]))

```

Comment:
This looks to be a copy of the example implementation, whereas this should be a stub with empty functions :)